impeller/renderer/backend/vulkan/context_vk_unittests.cc

```diff
@@ -222,5 +222,16 @@ TEST(ContextVKTest, WarmUpFunctionCreatesRenderPass) {
                         "vkCreateRenderPass") != functions->end());
 }
 
+TEST(ContextVKTest, FatalMissingValidations) {
```

Comment:
Is this compiled into `impeller_golden_tests`?  We should have one of these for that target too.

Reply:
This is just a check that the setting works. I've enabled this setting for playgrounds, so I should be able to confirm that removing any of the VVL config causes failures.

Reply:
What I meant is that we want to make sure that `impeller_golden_tests --gtest_filter=ContextVKTest.FatalMissingValidations` is executed too.  I suspect this will execute for `impeller_tests` but not `impeller_golden_tests`.

Reply:
That doesn't matter though, that isn't what is running the check.

Reply:
its just a test that the check works

Reply:
> its just a test that the check works

I know =)

> that isn't what is running the check.

I know, I want this test running in *both* executables.  In `impeller_unittests` and `impeller_golden_tests` we should fail if we don't have vulkan validation.

Reply:
I looked into this more. It doesn't seem possible for me to set up a playground where validations are requested but still misisng, without mocking out the loader. Did you have a different idea here?